### DISCUSSION TITLE
Migrating issues from the old repo

### DISCUSSION THREAD

**Discussion by TNorbury** [TinyCommunity/tinycolor2#6] [General]
Following up on https://github.com/TinyCommunity/tinycolor2/discussions/2#discussioncomment-917485, I'm thinking it might be a good idea to eventually migrate the issues/PRs over after a waiting period of sorts. Obviously wanna give people a chance to do it themselves, but it's hard if they'll see it or not. So perhaps after a week or so we copy the issues over, and for the PRs we can just create a fork of the PR creator's fork and open a PR on that, that way they'll still get credit for the commits and such.

Any thoughts?

**Comment by calvintam236** [comment]
https://github.com/TinyCommunity/tinycolor2/discussions/2#discussioncomment-917485

For those issues I didn’t comment are either fixed here or staled a long time. You can read throguh the old issues and leave a comment for those you think they should be migrated over.
